core/domain/html_cleaner.py

```diff
@@ -146,8 +146,53 @@ def get_rte_components(html_string):
     return components
 
 
+def html_escaper(html_data):
+    """This functions converts html to escaped format.
+
+    Args:
+        html_data: str. HTML string to be escaped.
+
+    Returns:
+        str. Escaped HTML string
+    """
+    replace_list = [
+        ('&', '&amp;'),
+        ('"', '&quot;'),
+        ('\'', '&#39;'),
+        ('<', '&lt;'),
+        ('>', '&gt;')
+    ]
+    for replace_tuple in replace_list:
+        html_data = html_data.replace(replace_tuple[0], replace_tuple[1])
+
+    return html_data
+
+
+def html_unescaper(html_data):
```

Comment:
def unescape_html(escaped_html_data)

Always start functions with verbs.

Reply:
Done! Got it!